### ISSUE TITLE
Fallout 4 Loot Version 0.13

### ISSUE THREAD

**Issue by Jadex611** [loot/loot#964]
loot cant seem to sort the plugins for Fallout 4 due to the old issue with pluginstxt.ini being read only but the problem is that it's not ticked to be read only but that's the error I'm getting in the debug so I suppose it must just be an issue with Loot hope you can check it out and see if I'm the only one with this problemo or not

**Comment by WrinklyNinja** [comment]
Please post your full debug log with debug logging enabled, see [How To Report Issues Helpfully](http://loot.github.io/docs/help/How-To-Report-Issues-Helpfully.html).

**Comment by Ghostwalker71** [comment]
I have been noticing this as well, It generally happens after using a mod manager (NMM or Wrye) to uninstall a mod and then install a newer versions of the same mod. Though I haven't been able to reproduce it effectively enough to send a report.
To get past the issue I cancel the loot process and close Loot, then open and close the mod manager without making any changes. then reopen loot and it will run properly, sorting and saving the plugin.txt file.
I have verified that the plugin text file is not marked as read only at any time.

**Comment by Darkcyde13** [comment]
I've been having a similar problem. At first LOOT said the Masterlist was missing and my log was as follows:-
```
[14:36:17.315583] [warning]: Chromium console message: Styling master document from stylesheets defined in HTML Imports is deprecated, and is planned to be removed in M65, around March 2018. Please refer to https://goo.gl/EGXzpw for possible migration paths.
[14:36:17.379364] [warning]: Blocking load of resource at https://fonts.googleapis.com/css?family=Roboto+Mono:400,700|Roboto:400,300,300italic,400italic,500,500italic,700,700italic
[14:36:18.825690] [warning]: Not a Git repository: C:\Users\Jason\AppData\Local\LOOT\Fallout4
[14:36:24.790794] [error]: Exception while executing query: boost::filesystem::rename: Access is denied: "E:\WinTemp\LOOT-Fallout4-d7b6-a51e-e6a2-68b5\.git", "C:\Users\Jason\AppData\Local\LOOT\Fallout4\.git"
[14:36:24.791576] [error]: Chromium console message from http://loot/ui/app.bundle.js at line 1: Error: Oh no, something went wrong! You can check your LOOTDebugLog.txt (you can get to it through the main menu) for more information.
    at onFailure (http://loot/ui/app.bundle.js:1:462892)
[14:36:28.873915] [error]: Exception while executing query: boost::filesystem::rename: Access is denied: "E:\WinTemp\LOOT-Fallout4-fb92-fbe9-5616-d79e\.git", "C:\Users\Jason\AppData\Local\LOOT\Fallout4\.git"
[14:36:28.874358] [error]: Chromium console message from http://loot/ui/app.bundle.js at line 1: Error: Oh no, something went wrong! You can check your LOOTDebugLog.txt (you can get to it through the main menu) for more information.
    at onFailure (http://loot/ui/app.bundle.js:1:462892)

```
This was before I read that I needed the debug turned on in settings. The Fallout4 folders on both C: and E: are full access, not read only.  It looks like the error says it's trying to open a file with no filename? '\.git'?

Anyway, after enabling the debug option in settings, LOOT says the Masterlist is 'Unkown, Git repository missing'. and I get the log as attached.
[LOOTDebugLog.txt](https://github.com/loot/loot/files/2066006/LOOTDebugLog.txt)

I've tried Ghostwalker71's suggestion of opening NMM (version 65.2) and closing without doing anything, and now LOOT refuses to run at all.

**Comment by Ghostwalker71** [comment]
I just ran 13.1 and had the same issue occur. Starting Loot and updating the master list ran without issue. I sorted the plugins and everything sorted into expected positions. upon trying to save the result it gave the generic uh-oh warning that indicated the plugin file is read only. I verified it was not read only.
Canceling the sort and closing loot then opening and closing the mod manager, without making any changes, then running loot again allowed it to progress and save the plugin text file.

**Comment by WrinklyNinja** [comment]
@Ghostwalker71 @Darkcyde13 please file separate issues for your problems, if they turn out to be duplicates then I can close them but until then let's not hijack @Jadex611's issue thread.

**Comment by Jadex611** [comment]
I mean 7 heads are better than one or something like that anyhow here's the error info sorry i should have assumed you needed that like i said before thoughi 100% have the plugins on edit not read only so thats not the problem

[07:26:35.016739] [warning]: Chromium console message: Styling master document from stylesheets defined in HTML Imports is deprecated, and is planned to be removed in M65, around March 2018. Please refer to https://goo.gl/EGXzpw for possible migration paths.
[07:26:35.685408] [warning]: Blocking load of resource at https://fonts.googleapis.com/css?family=Roboto+Mono:400,700|Roboto:400,300,300italic,400italic,500,500italic,700,700italic
[07:26:50.402652] [error]: libloadorder failed to set the load order. Details: The game's main master file must load first
[07:26:50.402973] [error]: Exception while executing query: libloadorder failed to set the load order. Details: The game's main master file must load first: Libloadorder error
[07:26:50.448883] [error]: Chromium console message from http://loot/ui/app.bundle.js at line 1: Error: Oh no, something went wrong! This is usually because "C:\Users\Jadex\AppData\Local\Fallout4\plugins.txt" is set to be read-only. If it is, unset it and try again. If it isn't, you can check your LOOTDebugLog.txt (you can get to it through the main menu) for more information.
    at onFailure (http://loot/ui/app.bundle.js:1:462892)

**Comment by WrinklyNinja** [comment]
Yep, duplicate of #956, this should be fixed in the 0.13.1 release.

**Comment by Jadex611** [comment]
nice
